src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: 'Announcing <model-viewer> 1.0'
+subhead: 3D models for your web page as easily as writing HTML.
+authors:
+  - rkochman
+  - joemedley
+description: |
+3D models are more relevant than ever. Retailers bring in-store shopping
+experiences to customers' homes. Museums let anyone see artifacts from anywhere.
+3D is difficult to do without a deep knowledge of 3D technologies or third-party
+hosting. <model-viewer> 1.0 makes these tasks as easy as writing HTML.
+date: 2020-08-20
+hero: hero.jpg
+alt: A 3d image of a shark.
+tags:
+  - blog
+  - 3d
+  - model-viewer
+---
+
+3D models are more relevant than ever. Retailers bring in-store shopping
+experiences to customers' homes. Museums are making 3D models of their artifacts
+available to everyone on the web. Unfortunately, it can be difficult to add a 3D
+model to a website in a way that provides a great user experience without a deep
+knowledge of 3D technologies or resorting to hosting 3D content on a third-party
+site. The `<model-viewer>` web component, [introduced in early
+2019](../model-viewer), seeks to
+make putting 3D models on the web as easy as writing a few lines of HTML. Since
+then, the team has been working to address feedback and requests from the
+community. The culmination of that work is `<model-viewer>` version 1.0.
+
+## What's new since last year?
+
+Version 1.0 includes built-in support for augmented reality (AR) on the web,
```

Comment:
1.1 is out now; should we update?

Reply:
I wonder if talking about 1.0 wouldn't be easier. Even if it's after 1.1 is out, everything here still applies, right?

Reply:
Yes, everything still applies. Mentioning v1.1 at the bottom sounds like a good compromise.